examples/flutter_gallery/lib/gallery/backdrop.dart

```diff
@@ -282,11 +282,11 @@ class _BackdropState extends State<Backdrop> with SingleTickerProviderStateMixin
             ),
           ),
           new Expanded(
-            child: new _TappableWhileStatusIs(
-              AnimationStatus.dismissed,
-              controller: _controller,
+            child: new Visibility(
               child: widget.backLayer,
-            ),
+              visible: _controller.status != AnimationStatus.completed,
+              maintainState: true,
```

Comment:
why do we need to maintain state?

Reply:
To maintain the menu scroll state so the new behavior matches the old behavior. I'm Ok to set it to false so the menu is always at the top when we open it.